pylops/signalprocessing/FFT.py

```diff
@@ -185,8 +189,12 @@ def _rmatvec(self, x):
             y = scipy.fft.ifft(x, n=self.nfft, axis=self.dir, **self._norm_kwargs)
         if self.norm is _FFTNorms.NONE:
             y *= self._scale
-        if self.nfft != self.dims[self.dir]:
+
+        if self.nfft > self.dims[self.dir]:
             y = np.take(y, np.arange(0, self.dims[self.dir]), axis=self.dir)
```

Comment:
I think this is a good opportunity to change all of these lines to the simpler:
```python
np.take(y, range(self.dims[self.dir]), axis=self.dir)
```

Reply:
Agreed!